interfaces/portal/karma.conf.js

```diff
@@ -0,0 +1,36 @@
+// Karma configuration file, see link for more information
+// https://karma-runner.github.io/1.0/config/configuration-file.html
```

Comment:
Less wrong, more specific:
```suggestion
// https://karma-runner.github.io/6.4/config/configuration-file.html
```

Might be wrong, more realistic, less to change/update:
```suggestion
// https://karma-runner.github.io/latest/config/configuration-file.html
```